installer-static.tpl.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+
+# This script is used to install the required packages for the application
+
+# Function for detect OS and set the package manager
+function detect_package_manager() {
+  ARCH=$(uname -m)
+
+  if yum --version > /dev/null 2>&1 || dnf --version > /dev/null 2>&1; then
+      echo "RhEL like"
+      install_yum_dnf
+  elif apt-get --version > /dev/null 2>&1; then
+      echo "Debian like"
+      install_apt
+  elif zypper --version > /dev/null 2>&1; then
+      echo "SuSe like"
+      install_zypper
+  else
+      echo "Unsupported OS"
+      exit 1
+  fi
+}
+
+install_apt ()
+{
+  apt-get update
+  apt-get install -y apt-transport-https curl gnupg
+
+  gpg_key_url="https://download.tarantool.org/tarantool/{{ rtype }}/series-{{ tarantool_version }}/gpgkey"
+  gpg_key_url_modules="https://download.tarantool.org/tarantool/release/modules/gpgkey"
+  apt_source_path="/etc/apt/sources.list.d/tarantool_{{ tarantool_version }}.list"
+
+  curl -L "${gpg_key_url}" | apt-key add -
+  curl -L "${gpg_key_url_modules}" | apt-key add -
+
+  echo "deb https://download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}/tarantool/{{ rtype }}/series-{{ tarantool_version }}/linux-deb/ static main" > ${apt_source_path}
+  echo "deb https://download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}/tarantool/release/modules/linux-deb static main" >> ${apt_source_path}
+
+  mkdir -p /etc/apt/preferences.d/
+  echo -e "Package: tarantool\nPin: origin download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}\nPin-Priority: 1001" > /etc/apt/preferences.d/tarantool
+  echo -e "\nPackage: tarantool-common\nPin: origin download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}\nPin-Priority: 1001" >> /etc/apt/preferences.d/tarantool
+  echo -e "\nPackage: tarantool-dev\nPin: origin download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}\nPin-Priority: 1001" >> /etc/apt/preferences.d/tarantool
+  echo "The repository is setup! Tarantool can now be installed."
+
+  apt-get update
+
+  echo
+  echo "Tarantool {{ tarantool_version }} is ready to be installed by 'apt-get -y install tarantool'"
+
+  if [[ ${FORCE_INSTALL_TARANTOOL:-False} = "True" ]]; then
```

Comment:
I think it is a wrong var name for such an activity. I suggest using the `TARANTOOL_AUTO_INSTALL` var.